pkg/api/vlabs/types.go

```diff
@@ -809,3 +822,16 @@ func (k *KubernetesConfig) IsRBACEnabled() bool {
 	}
 	return false
 }
+
+// IsFeatureEnabled returns true if a feature flag is on for the provided feature
+func (f *FeatureFlags) IsFeatureEnabled(feature string) bool {
+	if f != nil {
+		switch feature {
+		case "EnableIPv6DualStack":
```

Comment:
let's define the whole function here, not just the `EnableIPv6DualStack` scenario. Even if only the IPv6 flag is being used right now, this could lead to future bugs if developers assume that the function checks for all existing feature flags. Another option would be to rename the function to something more specific like `IsIPv6DualStackEnabled` to avoid confusion.

Reply:
Also, passing in the string is a leftover implementation from before the template refactor when we needed to pass in the arg in go templating. Since you're only using this function in `*.go` files, it could just be:

```
func(f *FeatureFlags) IsIPv6DualStackEnabled() bool {
    return f != nil && f. EnableIPv6DualStack
}
```